<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameTablePreview</class>
 <widget class="QFrame" name="GameTablePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>246</height>
   </size>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layer_bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>246</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="EntityWidget" name="bgcolor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>498</width>
      <height>246</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>498</width>
      <height>246</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>498</width>
      <height>246</height>
     </size>
    </property>
    <property name="backgroundColor">
     <color alpha="239">
      <red>32</red>
      <green>123</green>
      <blue>158</blue>
     </color>
    </property>
    <property name="respondable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="FramedEntity" name="framedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>498</width>
      <height>246</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>498</width>
      <height>246</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>498</width>
      <height>246</height>
     </size>
    </property>
    <property name="blending">
     <enum>E::Multiply</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/bgtest.png</pixmap>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>1000</width>
      <height>624</height>
     </size>
    </property>
    <property name="aspectRatioMode">
     <enum>Qt::KeepAspectRatioByExpanding</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layer_table" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>246</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <widget class="FramedEntity" name="tabletexture">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>251</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>371</width>
      <height>251</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/table1.png</pixmap>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>830</width>
      <height>584</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>25</y>
      <width>268</width>
      <height>164</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
border-radius: 120px 82px;
}</string>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_1">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>70</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>12</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_2">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>70</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>33</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_3">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>70</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>25</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_4">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>70</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>46</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>tabletexture</zorder>
   <zorder>tablecard_0</zorder>
   <zorder>tablecard_1</zorder>
   <zorder>tablecard_2</zorder>
   <zorder>tablecard_3</zorder>
   <zorder>tablecard_4</zorder>
  </widget>
  <widget class="QWidget" name="layer_players" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>246</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>498</width>
     <height>246</height>
    </size>
   </property>
   <widget class="FramedEntity" name="tablecard_5">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>134</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>2</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecard_6">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>134</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/cards13x4_1.png</pixmap>
    </property>
    <property name="frameIndex">
     <number>15</number>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>52</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>163</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fafafa;
    
	font: 775 9px &quot;Open Sans&quot;;
border-image: url(:/textures/game/Userbox.png);
}</string>
    </property>
    <property name="text">
     <string notr="true">nickname</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/icon_0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>53</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fafafa;
    
	font: 775 9px &quot;Open Sans&quot;;
border-image: url(:/textures/game/Userbox.png);
}</string>
    </property>
    <property name="text">
     <string notr="true">nickname</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/icon_0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecardback_0">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>29</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/shirt_card.png</pixmap>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>58</width>
      <height>84</height>
     </size>
    </property>
   </widget>
   <widget class="FramedEntity" name="tablecardback_1">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>31</y>
      <width>35</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>50</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/textures/game/shirt_card.png</pixmap>
    </property>
    <property name="sizeOfFrame">
     <size>
      <width>58</width>
      <height>84</height>
     </size>
    </property>
   </widget>
   <zorder>tablecardback_1</zorder>
   <zorder>tablecardback_0</zorder>
   <zorder>tablecard_5</zorder>
   <zorder>tablecard_6</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QWidget" name="layer_controls" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>498</width>
     <height>246</height>
    </rect>
   </property>
  </widget>
  <zorder>layer_bg</zorder>
  <zorder>layer_table</zorder>
  <zorder>layer_controls</zorder>
  <zorder>layer_players</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FramedEntity</class>
   <extends>EntityWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
